README.md

```diff
@@ -151,6 +153,9 @@ The Playwright MCP server can be configured using a JSON configuration file. Her
   // Directory for output files
   outputDir?: string;
 
+  // List of Playwright Glob URL patterns (https://playwright.dev/docs/network#glob-url-patterns) to allow the browser to request. Default is to allow all.
+  requestPatternAllowlist?: string[];
```

Comment:
- fold under `network`. `network.allowedOrigins`
- blocklist is missing in docs